Foundation/NSURLProtocol.swift

```diff
@@ -148,7 +148,8 @@ public protocol URLProtocolClient : NSObjectProtocol {
 internal class _ProtocolClient : NSObject, URLProtocolClient {
 
     func urlProtocol(_ protocol: URLProtocol, didReceive response: URLResponse, cacheStoragePolicy policy: URLCache.StoragePolicy) {
-        `protocol`.task?.response = response
+	guard let task = `protocol`.task else { fatalError() }
```

Comment:
This is a breaking change in behaviour. Previously it would be a no-operation, now it will crash call. It's not clear whether this is beneficial and certainly doesn't support the message that this PR was filed with.